tests/api/test_security.py

```diff
@@ -0,0 +1,85 @@
+import requests
+import os
+
+
+class TestPOST(object):
+    def setup(self):
+        # app['TESTING'] = True
+        # self.app = app.test_client()
+        pass
+
+    def test_main_page(self, flask_api_url):
+        response = requests.get(flask_api_url + '/')
+        assert response.status_code == 200
+
+    def test_login_fake_user(self, flask_api_url):
+        session = requests.Session()
+        response = session.post(flask_api_url + '/api/users/login',
+                                data=dict(email='user1', password='password'))
+
+        assert response.reason == 'FORBIDDEN'
```

Comment:
as mentioned above, checking response reasons is pretty pointless if you already check status codes.